### PR TITLE
Fix lspci output change issue for vsock device

### PR DESCRIPTION
**Pull request by qcheng-redhat**
Fix lspci output change issue for vsock device when machine_type=q35.

ID: 1916620

Signed-off-by: Qinghua Cheng <qcheng@redhat.com>

### CODE REVIEW

**Comment by qcheng-redhat** [comment]
Test results:

For rhel.8.4.0:
python3 ConfigTest.py --guestname=RHEL.8.4.0 --testcase=vsock_test.hotplug --clone=no --machine=q35
(1/1) Host_RHEL.m8.u4.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.4.0.x86_64.io-github-autotest-qemu.vsock_test.hotplug.q35: PASS (169.17 s)

python3 ConfigTest.py --guestname=RHEL.8.4.0 --testcase=vsock_test.hotplug --clone=no
(1/1) Host_RHEL.m8.u4.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.4.0.x86_64.io-github-autotest-qemu.vsock_test.hotplug: PASS (139.21 s)


For rhel.8.2.1 
python3 ConfigTest.py --guestname=RHEL.8.2.1 --testcase=vsock_test.hotplug --clone=no --machine=q35
(1/1) Host_RHEL.m8.u4.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.2.1.x86_64.io-github-autotest-qemu.vsock_test.hotplug.q35: PASS (162.69 s)

python3 ConfigTest.py --guestname=RHEL.8.2.1 --testcase=vsock_test.hotplug --clone=no
(1/1) Host_RHEL.m8.u4.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.2.1.x86_64.io-github-autotest-qemu.vsock_test.hotplug: PASS (154.84 s)

**Comment by qcheng-redhat** [comment]
@vivianQizhu Can you help review this patch? Thanks.

File: qemu/tests/cfg/vsock.cfg [hunk 1]
```diff
@@ -12,5 +12,8 @@
             vsocks = ''
             addr_pattern = '\d\d:\d\d\.\d'
             device_pattern = 'Communication controller: Red Hat.* Device'
+            q35:
+              RHEL.8.3, RHEL.8.4:
+                device_pattern = 'Communication controller: Red Hat.* Virtio socket'
```
Comment (vivianQizhu) [inline qemu/tests/cfg/vsock.cfg:17]: This way you will always need to append new version when it comes out.
```suggestion
            device_pattern = 'Communication controller: Red Hat.* Device'
            q35:
                device_pattern = 'Communication controller: Red Hat.* Virtio socket'
                RHEL.7, RHEL.8.1, RHEL.8.2:
                    device_pattern = 'Communication controller: Red Hat.* Device'
```

Comment (qcheng-redhat) [reply qemu/tests/cfg/vsock.cfg]: Updated. Thank.

Comment (qcheng-redhat) [reply qemu/tests/cfg/vsock.cfg]: python3 ConfigTest.py --guestname=RHEL.8.4.0 --testcase=vsock_test.hotplug --clone=no --machine=q35
(1/1) Host_RHEL.m8.u4.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.4.0.x86_64.io-github-autotest-qemu.vsock_test.hotplug.q35: PASS (169.00 s)

python3 ConfigTest.py --guestname=RHEL.8.2.1 --testcase=vsock_test.hotplug --clone=no --machine=q35
(1/1) Host_RHEL.m8.u4.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.2.1.x86_64.io-github-autotest-qemu.vsock_test.hotplug.q35: PASS (161.31 s)

**Review by vivianQizhu — APPROVED** [review]

**Pull request merged**